src/registrar/assets/sass/_theme/_base.scss

```diff
@@ -190,7 +190,7 @@ abbr[title] {
     svg.usa-icon {
       color: #{$dhs-red};
     }
-    div.readonly-field {
+    div.input-with-edit-button__readonly-field {
```

Comment:
This strips away the red coloring for other readonly fields.  Did we want to do this?

Reply:
It's not really an issue...The only thing is that when we get specific like this we rely more and more on several "red" settings to basically stay in sync, which seems prone to breakages.  We do this alot already in our scss architecture, though, so I will leave it.  I've noticed that we are gradually refactoring our scss as we go so it shouldn't be too much of a problem.